runtime/queries/slint/highlights.scm

```diff
@@ -118,8 +148,7 @@
  "*="
  "/="
  "?"
-
  "=>"
  ] @operator
 
-(ternary_expression [":" "?"] @keyword.control.conditional)
\ No newline at end of file
+(ternary_expression [":" "?"] @conditional)
```

Comment:
Helix uses different captures for highlighting than neovim - https://docs.helix-editor.com/master/themes.html#syntax-highlighting- so this change should be undone

There are some other non-existent scopes that should be changed too, for example `@float` should become `@constant.numeric.float`